lms/static/scripts/frontend_apps/components/FilePickerApp.js

```diff
@@ -1,10 +1,11 @@
-import { createElement } from 'preact';
+import { Fragment, createElement } from 'preact';
```

Comment:
Everything in this file is just me trying to make it work with as less changes as possible so it's wrong in many ways.